plot_types/basic/stackplot.py

```diff
@@ -0,0 +1,28 @@
+"""
+===============================
+stackplot(x, y)
+===============================
+See `~matplotlib.axes.Axes.stackplot`
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+plt.style.use('_mpl-gallery')
+
+# make data
+x = np.arange(0, 10, 2)
+ay = [1, 1.25, 2, 2.75, 3]
+by = [1, 1, 1, 1, 1]
+cy = [2, 1, 2, 1, 2]
+y = np.vstack([ay, by, cy])
+labels = ['a', 'b', 'c']
+# plot
+fig, ax = plt.subplots()
+
+ax.stackplot(x, y, labels=labels)
```

Comment:
I'm not in favour of adding this, but if others support adding it, please change the colors to be shades of the default blue to match the other plots.

Reply:
I thought that came w/ 'plt.style.use('_mpl-gallery') so I don't know why that isn't working.

Reply:
And I think its' cause prop_cycle is commented out in the style files https://github.com/matplotlib/matplotlib/blob/main/lib/matplotlib/mpl-data/stylelib/_mpl-gallery.mplstyle but then I'm confused how the other functions work so gonna go track that down